offer-manager-domain/pom.xml

```diff
@@ -1,43 +1,47 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xmlns="http://maven.apache.org/POM/4.0.0"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <artifactId>offer-manager-domain</artifactId>
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <artifactId>offer-manager</artifactId>
-    <groupId>life.qbic</groupId>
-    <version>1.0.0-alpha.4-SNAPSHOT</version>
-  </parent>
-  <dependencies>
-      <dependency>
-          <groupId>life.qbic</groupId>
-          <artifactId>data-model-lib</artifactId>
-      </dependency>
-      <dependency>
-          <groupId>log4j</groupId>
-          <artifactId>log4j</artifactId>
-          <version>1.2.17</version>
-      </dependency>
-  </dependencies>
-  <build>
-    <plugins>
-    <plugin>
-    <groupId>org.codehaus.gmavenplus</groupId>
-    <artifactId>gmavenplus-plugin</artifactId>
-      <configuration>
-        <groovyDocOutputDirectory>${project.build.directory}/site/gapidocs</groovyDocOutputDirectory>
-        <testGroovyDocOutputDirectory>${project.build.directory}/site/testgapidocs</testGroovyDocOutputDirectory>
-      </configuration>
-    </plugin>
-    </plugins>
-  </build>
-  <reporting>
-    <plugins>
-      <plugin>
-        <groupId>life.qbic</groupId>
-        <artifactId>groovydoc-maven-plugin</artifactId>
-      </plugin>
-    </plugins>
-  </reporting>
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<artifactId>offer-manager-domain</artifactId>
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>offer-manager</artifactId>
+		<groupId>life.qbic</groupId>
+		<version>1.0.0-alpha.4-SNAPSHOT</version>
+	</parent>
+	<dependencies>
+		<dependency>
+			<groupId>life.qbic</groupId>
+			<artifactId>data-model-lib</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>life.qbic</groupId>
+			<artifactId>openbis-client-lib</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>log4j</groupId>
+			<artifactId>log4j</artifactId>
+			<version>1.2.17</version>
+		</dependency>
+	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.gmavenplus</groupId>
+				<artifactId>gmavenplus-plugin</artifactId>
```

Comment:
indentation with tabs and not with spaces?

Reply:
this is what formatting with google style does for me...sorry about that. if you want you could try to fix it with your IDE

Reply:
weird